modules/images-other-jenkins-agent-images.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * images/using_images/images-other-jenkins-agent.adoc
+
+[id="images-other-jenkins-agent-images_{context}"]
+= Jenkins agent images
+
+The {product-title} Jenkins agent images come in two varieties:
+
+*RHEL 7 Based Images*
+
+RHEL 7 images are available through the Red Hat Registry:
+
+----
+$ docker pull registry.redhat.io/openshift3/jenkins-slave-base-rhel7
+$ docker pull registry.redhat.io/openshift3/jenkins-slave-maven-rhel7
+$ docker pull registry.redhat.io/openshift3/jenkins-slave-nodejs-rhel7
```

Comment:
The slave-maven and slave-nodejs are being deprecated as part of v3.10 of OpenShift. Additionally, development of these images will cease as of v3.10. And they are not used in the v4.1 development cycle.

Reply:
Above 3 slave images are deprecated.